Moonscraper Chart Editor/Assets/Scripts/Game/UI/Menus/SongPropertiesPanelController.cs

```diff
@@ -320,6 +346,43 @@ public void RefreshAllAudioStreams()
         StartCoroutine(_RefreshAllAudioStreams());
     }
 
+    public void LoadCloneHeroAudioFromDirectory()
+    {
+        var currentDir = ChartEditor.Instance.lastLoadedFile;
+        if (string.IsNullOrWhiteSpace(currentDir))
+        {
+            FileExplorer.OpenFolderPanel(out currentDir);
```

Comment:
This needs to handle the case of the user cancelling the open folder panel prompt. This should early return upon `FileExplorer.OpenFolderPanel` returning false.